views/src/main/java/io/micronaut/views/thymeleaf/LinkBuilder.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2017-2019 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.micronaut.views.thymeleaf;
+
+import io.micronaut.http.HttpRequest;
+import org.thymeleaf.context.IExpressionContext;
+import org.thymeleaf.exceptions.TemplateProcessingException;
+import org.thymeleaf.linkbuilder.StandardLinkBuilder;
+
+import java.util.Map;
+
+/**
+ * Implementation of {@link org.thymeleaf.linkbuilder.ILinkBuilder} interface for {@link WebContext}
+ * support.
+ *
+ * @author Semyon Gashchenko
+ */
+class LinkBuilder extends StandardLinkBuilder {
```

Comment:
This should probably be a bean

Reply:
Can't think of a good use outside TemplateEngine configuring.

Reply:
It doesn't have to have a good use outside the template engine configuring to be a bean